src/test/librbd/test_mock_ManagedLock.cc

```diff
@@ -393,6 +393,27 @@ TEST_F(TestMockManagedLock, AcquireLockBlocklist) {
   ASSERT_EQ(0, when_shut_down(managed_lock));
 }
 
+TEST_F(TestMockManagedLock, SendAcquireLockBlocklist) {
+  librbd::ImageCtx *ictx;
+  ASSERT_EQ(0, open_image(m_image_name, &ictx));
+
+  MockManagedLockImageCtx mock_image_ctx(*ictx);
+  MockMockManagedLock managed_lock(ictx->md_ctx, *ictx->asio_engine,
+                               ictx->header_oid, mock_image_ctx.image_watcher,
+                               librbd::managed_lock::EXCLUSIVE, true, 0);
+
+  InSequence seq;
+
+  expect_get_watch_handle(*mock_image_ctx.image_watcher, 0);
+  C_SaferCond acquire_ctx;
+
+  EXPECT_CALL(*mock_image_ctx.image_watcher, is_blocklisted()).WillOnce(Return(true));
```

Comment:
Wrap this into `expect_is_blocklisted` similar to other `expect_*` methods.  It would need to be used in `AttemptReacquireBlocklistedLock` and `ShutDownWhileWaiting` tests to avoid
```
GMOCK WARNING:
Uninteresting mock function call - returning default value.
    Function call: is_blocklisted()
          Returns: false
```
warnings.

Reply:
Done